src/v3/src/hooks/useOnValidate.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { StateUpdater, useCallback } from 'preact/hooks';
+
+import { getMessage } from '../../../v2/ion/i18nTransformer';
+import { useWidgetContext } from '../contexts';
+import { FieldElement } from '../types';
+
+export const useOnValidate = (uischema: FieldElement) => {
```

Comment:
is this hook only handling validation for field focus related validation? maybe give it a specific name to differentiate with form submission validation.

Reply:
Yep, exactly, will update to reflect.